modules/javafx.graphics/src/main/java/javafx/print/JobSettings.java

```diff
@@ -462,6 +462,98 @@ public void setJobName(String name) {
     }
     ///////////////////////  END JOBNAME /////////////////////
 
+    ///////////////////////  START OUTPUTFILE /////////////////////
+
+    private SimpleStringProperty outputFile;
+
+    /**
+     * A {@code StringProperty} representing the
+     * name of a filesystem file, to which the platform printer
+     * driver should spool the rendered print data.
+     * <p>
+     * Applications can use this to programmatically request print-to-file
+     * behavior where the native print system is capable of spooling the
+     * output to a filesystem file, rather than the printer device.
+     * <p>
+     * This is often useful where the printer driver generates a format
+     * such as Postscript or PDF, and the application intends to distribute
+     * the result instead of printing it, or for some other reason the
+     * application does not want physical media (paper) emitted by the printer.
```

Comment:
Very minor: maybe consider combining the first three paragraphs into a single paragraph?

Reply:
well .. I usually like a short paragraph that succinctly says what it does as the first paragraph

Anyway I've re-read all this and I prefer it as it is.

Reply:
ok